charts/wadm/values.yaml

```diff
@@ -62,8 +63,8 @@ securityContext:
   seccompProfile:
     type: "RuntimeDefault"
 
-
-resources: {}
+resources:
+  {}
```

Comment:
Can you revert this change, I'm pretty sure we want to keep it the way it was.

Reply:
no problem. heads up this is literally the output from yaml linter